css/block.css

```diff
@@ -156,6 +158,84 @@ wb-context[closed] > wb-contains{
     overflow: hidden;
 }
 
+
+wb-value[type=number] > input{
```

Comment:
Right, I was trying that, but for whatever reasons if I didn't redefine the colour and the padding each time, the input box would be the same colour as the block (that tan-ish colour). Any insight would be appreciated!